### ISSUE TITLE
Show alt text not working for images in Notify popups

### ISSUE THREAD

**Issue by swashbuck** [cgkineo/adapt-devtools#119]
### Subject of the issue
The "show alt text" feature does not work for images in Notify (ex. Hot Graphic or Hot Grid popups). Specifically, there is not a plus icon that shows what the alt text is. May also affect images in drawer.

<img width="347" height="389" alt="Image" src="https://github.com/user-attachments/assets/1de53365-36c0-4123-8bc9-6275f29d8621" />
